test/py37/negative_cases/incompatible_lock_options/pyproject.toml

```diff
@@ -0,0 +1,24 @@
+[tool.poetry]
+name = "blixexample"
+version = "0.1.0"
+description = "Example Python package used to test poeblix"
+authors = ["spoorn <spookump@gmail.com>"]
+
+[tool.poetry.dependencies]
+python = "^3.9"
```

Comment:
My bad, missed this in my last review.  You'll need to change these for the corresponding python version.  Like so: https://github.com/spoorn/poeblix/blob/main/test/py37/positive_cases/no_lock/pyproject.toml#L8.  And then you'll probably need to regenerate the poetry.lock file for the tests.

Sorry it's a bit of a pain, but I've caught issues with py37 and py38 with these tests in the past so I think it's useful

Reply:
I've just figured out that. By the way, I'm uploading the wheel package because you uploaded on the other examples but I don't think that's necessary because wheel files are created by the tests. Is there any reason for that?

Reply:
You may be able to just copy the poetry.lock and pyproject.toml from the other test files in the `py37` and `py38` directories, since I don't think you are creating different poetry files for the additional tests

Reply:
> I've just figured out that. By the way, I'm uploading the wheel package because you uploaded on the other examples but I don't think that's necessary because wheel files are created by the tests. Is there any reason for that?

It's not necessary for this test, but some of the other tests don't require a `poetry blixbuild` so I pregenerated the wheel file that's expected.  I haven't added the `dist` directories to `.gitignore` yet

Reply:
Fixed.